basics.md

```diff
@@ -288,19 +318,27 @@ from IPython.display import SVG
 SVG(ts.draw_svg(y_axis=True, y_gridlines=True, time_scale="rank", style=".mut {display: none}"))
 ```
 
+Note that all these trees show strictly bifurcating splits, but this does not need to be
+the case for tree sequences in general. In particular, where tree sequences have been
+created by the multiple-merger coalecent process, or in tree
+sequences that have been inferred from real data, it is possible to have a parent node
+with 3 or more children in a particular tree. Such multifucations are known as
```

Comment:
multifurcation is spelled wrongly, although it's a hideous bit of jargon anyway IMO. I don't think we lose much by deleting the last sentence.

Reply:
Good spot. I think it is worth mentioning "polytomy" (because we're likely to use it again in other tutes, especially if we write an inference one), but I agree about "multifurcation". How about simply putting "(this is known as a polytomy)" in brackets, or something similar?